app/views/task-list.html

```diff
@@ -80,6 +80,22 @@ <h2 class="app-task-list__section">
               </span>
               <strong class="govuk-tag app-task-list__task-completed" id="upload-a-photo-completed">Completed</strong>
             </li>
+            <li class="app-task-list__item">
+              <span class="app-task-list__task-name">
+                <a class="govuk-link" href="/no-permission" aria-describedby="list-medication-completed">
+                  List medication
+                </a>
+              </span>
+              <strong class="govuk-tag app-task-list__task-completed" id="list-treatments-completed">Completed</strong>
```

Comment:
This should have the id `list-medication-completed` instead of `list-treatments-completed`